openreview/tools.py

```diff
@@ -366,12 +355,22 @@ def capitalize_title(title):
         else:
             authors = ' and '.join(note.content['authors'])
 
-    bibtex_title = capitalize_title(note.content['title'])
+    u = UnicodeToLatexEncoder(
+        conversion_rules=[
+            UnicodeToLatexConversionRule(
+                rule_type=RULE_REGEX,
+                rule=[
+                    (re.compile(r'[A-Z]{2,}'), r'{\g<0>}')
+                ]),
+            'defaults'
+        ]
+    )
+    bibtex_title = u.unicode_to_latex(note.content['title'])
```

Comment:
why do you apply this to the title only and the other fields still use `utf8tolatex`, it seems that `utf8tolatex` is deprecated: https://pylatexenc.readthedocs.io/en/latest/latexencode/

Reply:
because title is transformed with capitalize_title fn so that it will be transformed wrongly, the other fields won't have this problem.

utf8tolatex will work fine and is clearer otherwise we will have to construct two separate encoders for title and other fields